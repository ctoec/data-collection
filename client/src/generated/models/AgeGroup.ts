/* tslint:disable */
/* eslint-disable */
/**
 * data-collection
 * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
 *
 * The version of the OpenAPI document: 0.0.1
 *
 *
 * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
 * https://openapi-generator.tech
 * Do not edit the class manually.
 */

/**
 *
 * @export
 * @enum {string}
 */
export enum AgeGroup {
  InfantToddler = 'InfantToddler',
  Preschool = 'Preschool',
  SchoolAge = 'SchoolAge',
}

export function AgeGroupFromJSON(json: any): AgeGroup {
  return AgeGroupFromJSONTyped(json, false);
}

export function AgeGroupFromJSONTyped(
  json: any,
  ignoreDiscriminator: boolean
): AgeGroup {
  return json as AgeGroup;
}

export function AgeGroupToJSON(value?: AgeGroup | null): any {
  return value as any;
}
