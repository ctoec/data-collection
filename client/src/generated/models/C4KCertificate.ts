/* tslint:disable */
/* eslint-disable */
/**
 * data-collection
 * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
 *
 * The version of the OpenAPI document: 0.0.1
 * 
 *
 * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
 * https://openapi-generator.tech
 * Do not edit the class manually.
 */

import { exists, mapValues } from '../runtime';
import {
    Child,
    ChildFromJSON,
    ChildFromJSONTyped,
    ChildToJSON,
} from './';

/**
 * 
 * @export
 * @interface C4KCertificate
 */
export interface C4KCertificate {
    /**
     * 
     * @type {number}
     * @memberof C4KCertificate
     */
    id: number;
    /**
     * 
     * @type {string}
     * @memberof C4KCertificate
     */
    childId: string;
    /**
     * 
     * @type {Child}
     * @memberof C4KCertificate
     */
    child?: Child;
    /**
     * 
     * @type {Date}
     * @memberof C4KCertificate
     */
    startDate?: Date;
    /**
     * 
     * @type {Date}
     * @memberof C4KCertificate
     */
    endDate?: Date;
}

export function C4KCertificateFromJSON(json: any): C4KCertificate {
    return C4KCertificateFromJSONTyped(json, false);
}

export function C4KCertificateFromJSONTyped(json: any, ignoreDiscriminator: boolean): C4KCertificate {
    if ((json === undefined) || (json === null)) {
        return json;
    }
    return {
        
        'id': json['id'],
        'childId': json['childId'],
        'child': !exists(json, 'child') ? undefined : ChildFromJSON(json['child']),
        'startDate': !exists(json, 'startDate') ? undefined : (new Date(json['startDate'])),
        'endDate': !exists(json, 'endDate') ? undefined : (new Date(json['endDate'])),
    };
}

export function C4KCertificateToJSON(value?: C4KCertificate | null): any {
    if (value === undefined) {
        return undefined;
    }
    if (value === null) {
        return null;
    }
    return {
        
        'id': value.id,
        'childId': value.childId,
        'child': ChildToJSON(value.child),
        'startDate': value.startDate === undefined ? undefined : (value.startDate.toISOString()),
        'endDate': value.endDate === undefined ? undefined : (value.endDate.toISOString()),
    };
}


