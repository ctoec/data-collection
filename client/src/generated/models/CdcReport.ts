/* tslint:disable */
/* eslint-disable */
/**
 * data-collection
 * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
 *
 * The version of the OpenAPI document: 0.0.1
 * 
 *
 * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
 * https://openapi-generator.tech
 * Do not edit the class manually.
 */

import { exists, mapValues } from '../runtime';
import {
    Enrollment,
    EnrollmentFromJSON,
    EnrollmentFromJSONTyped,
    EnrollmentToJSON,
    FundingSource,
    FundingSourceFromJSON,
    FundingSourceFromJSONTyped,
    FundingSourceToJSON,
    FundingTimeSplitUtilization,
    FundingTimeSplitUtilizationFromJSON,
    FundingTimeSplitUtilizationFromJSONTyped,
    FundingTimeSplitUtilizationToJSON,
    Organization,
    OrganizationFromJSON,
    OrganizationFromJSONTyped,
    OrganizationToJSON,
    ReportingPeriod,
    ReportingPeriodFromJSON,
    ReportingPeriodFromJSONTyped,
    ReportingPeriodToJSON,
} from './';

/**
 * 
 * @export
 * @interface CdcReport
 */
export interface CdcReport {
    /**
     * 
     * @type {boolean}
     * @memberof CdcReport
     */
    accredited: boolean;
    /**
     * 
     * @type {number}
     * @memberof CdcReport
     */
    c4KRevenue: number;
    /**
     * 
     * @type {boolean}
     * @memberof CdcReport
     */
    retroactiveC4KRevenue?: boolean;
    /**
     * 
     * @type {number}
     * @memberof CdcReport
     */
    familyFeesRevenue: number;
    /**
     * 
     * @type {string}
     * @memberof CdcReport
     */
    comment?: string;
    /**
     * 
     * @type {Array<FundingTimeSplitUtilization>}
     * @memberof CdcReport
     */
    timeSplitUtilizations?: Array<FundingTimeSplitUtilization>;
    /**
     * 
     * @type {number}
     * @memberof CdcReport
     */
    organizationId: number;
    /**
     * 
     * @type {Organization}
     * @memberof CdcReport
     */
    organization?: Organization;
    /**
     * 
     * @type {number}
     * @memberof CdcReport
     */
    id: number;
    /**
     * 
     * @type {FundingSource}
     * @memberof CdcReport
     */
    type?: FundingSource;
    /**
     * 
     * @type {number}
     * @memberof CdcReport
     */
    reportingPeriodId: number;
    /**
     * 
     * @type {ReportingPeriod}
     * @memberof CdcReport
     */
    reportingPeriod: ReportingPeriod;
    /**
     * 
     * @type {Date}
     * @memberof CdcReport
     */
    submittedAt?: Date;
    /**
     * 
     * @type {Array<Enrollment>}
     * @memberof CdcReport
     */
    enrollments?: Array<Enrollment>;
}

export function CdcReportFromJSON(json: any): CdcReport {
    return CdcReportFromJSONTyped(json, false);
}

export function CdcReportFromJSONTyped(json: any, ignoreDiscriminator: boolean): CdcReport {
    if ((json === undefined) || (json === null)) {
        return json;
    }
    return {
        
        'accredited': json['accredited'],
        'c4KRevenue': json['c4KRevenue'],
        'retroactiveC4KRevenue': !exists(json, 'retroactiveC4KRevenue') ? undefined : json['retroactiveC4KRevenue'],
        'familyFeesRevenue': json['familyFeesRevenue'],
        'comment': !exists(json, 'comment') ? undefined : json['comment'],
        'timeSplitUtilizations': !exists(json, 'timeSplitUtilizations') ? undefined : ((json['timeSplitUtilizations'] as Array<any>).map(FundingTimeSplitUtilizationFromJSON)),
        'organizationId': json['organizationId'],
        'organization': !exists(json, 'organization') ? undefined : OrganizationFromJSON(json['organization']),
        'id': json['id'],
        'type': !exists(json, 'type') ? undefined : FundingSourceFromJSON(json['type']),
        'reportingPeriodId': json['reportingPeriodId'],
        'reportingPeriod': ReportingPeriodFromJSON(json['reportingPeriod']),
        'submittedAt': !exists(json, 'submittedAt') ? undefined : (new Date(json['submittedAt'])),
        'enrollments': !exists(json, 'enrollments') ? undefined : ((json['enrollments'] as Array<any>).map(EnrollmentFromJSON)),
    };
}

export function CdcReportToJSON(value?: CdcReport | null): any {
    if (value === undefined) {
        return undefined;
    }
    if (value === null) {
        return null;
    }
    return {
        
        'accredited': value.accredited,
        'c4KRevenue': value.c4KRevenue,
        'retroactiveC4KRevenue': value.retroactiveC4KRevenue,
        'familyFeesRevenue': value.familyFeesRevenue,
        'comment': value.comment,
        'timeSplitUtilizations': value.timeSplitUtilizations === undefined ? undefined : ((value.timeSplitUtilizations as Array<any>).map(FundingTimeSplitUtilizationToJSON)),
        'organizationId': value.organizationId,
        'organization': OrganizationToJSON(value.organization),
        'id': value.id,
        'type': FundingSourceToJSON(value.type),
        'reportingPeriodId': value.reportingPeriodId,
        'reportingPeriod': ReportingPeriodToJSON(value.reportingPeriod),
        'submittedAt': value.submittedAt === undefined ? undefined : (value.submittedAt.toISOString()),
        'enrollments': value.enrollments === undefined ? undefined : ((value.enrollments as Array<any>).map(EnrollmentToJSON)),
    };
}


