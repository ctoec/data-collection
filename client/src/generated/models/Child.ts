/* tslint:disable */
/* eslint-disable */
/**
 * data-collection
 * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
 *
 * The version of the OpenAPI document: 0.0.1
 *
 *
 * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
 * https://openapi-generator.tech
 * Do not edit the class manually.
 */

import { exists, mapValues } from '../runtime';
import {
  Enrollment,
  EnrollmentFromJSON,
  EnrollmentFromJSONTyped,
  EnrollmentToJSON,
  Family,
  FamilyFromJSON,
  FamilyFromJSONTyped,
  FamilyToJSON,
  Gender,
  GenderFromJSON,
  GenderFromJSONTyped,
  GenderToJSON,
  Organization,
  OrganizationFromJSON,
  OrganizationFromJSONTyped,
  OrganizationToJSON,
  UpdateMetaData,
  UpdateMetaDataFromJSON,
  UpdateMetaDataFromJSONTyped,
  UpdateMetaDataToJSON,
} from './';

/**
 *
 * @export
 * @interface Child
 */
export interface Child {
  /**
   *
   * @type {string}
   * @memberof Child
   */
  id: string;
  /**
   *
   * @type {string}
   * @memberof Child
   */
  sasid?: string;
  /**
   *
   * @type {string}
   * @memberof Child
   */
  firstName: string;
  /**
   *
   * @type {string}
   * @memberof Child
   */
  middleName?: string;
  /**
   *
   * @type {string}
   * @memberof Child
   */
  lastName: string;
  /**
   *
   * @type {string}
   * @memberof Child
   */
  suffix?: string;
  /**
   *
   * @type {Date}
   * @memberof Child
   */
  birthdate?: Date;
  /**
   *
   * @type {string}
   * @memberof Child
   */
  birthTown?: string;
  /**
   *
   * @type {string}
   * @memberof Child
   */
  birthState?: string;
  /**
   *
   * @type {string}
   * @memberof Child
   */
  birthCertificateId?: string;
  /**
   *
   * @type {boolean}
   * @memberof Child
   */
  americanIndianOrAlaskaNative?: boolean;
  /**
   *
   * @type {boolean}
   * @memberof Child
   */
  asian?: boolean;
  /**
   *
   * @type {boolean}
   * @memberof Child
   */
  blackOrAfricanAmerican?: boolean;
  /**
   *
   * @type {boolean}
   * @memberof Child
   */
  nativeHawaiianOrPacificIslander?: boolean;
  /**
   *
   * @type {boolean}
   * @memberof Child
   */
  white?: boolean;
  /**
   *
   * @type {boolean}
   * @memberof Child
   */
  hispanicOrLatinxEthnicity?: boolean;
  /**
   *
   * @type {Gender}
   * @memberof Child
   */
  gender?: Gender;
  /**
   *
   * @type {boolean}
   * @memberof Child
   */
  foster?: boolean;
  /**
   *
   * @type {boolean}
   * @memberof Child
   */
  recievesC4K?: boolean;
  /**
   *
   * @type {Family}
   * @memberof Child
   */
  family: Family;
  /**
   *
   * @type {Organization}
   * @memberof Child
   */
  organization?: Organization;
  /**
   *
   * @type {Array<Enrollment>}
   * @memberof Child
   */
  enrollments?: Array<Enrollment>;
  /**
   *
   * @type {UpdateMetaData}
   * @memberof Child
   */
  updateMetaData?: UpdateMetaData;
}

export function ChildFromJSON(json: any): Child {
  return ChildFromJSONTyped(json, false);
}

export function ChildFromJSONTyped(
  json: any,
  ignoreDiscriminator: boolean
): Child {
  if (json === undefined || json === null) {
    return json;
  }
  return {
    id: json['id'],
    sasid: !exists(json, 'sasid') ? undefined : json['sasid'],
    firstName: json['firstName'],
    middleName: !exists(json, 'middleName') ? undefined : json['middleName'],
    lastName: json['lastName'],
    suffix: !exists(json, 'suffix') ? undefined : json['suffix'],
    birthdate: !exists(json, 'birthdate')
      ? undefined
      : new Date(json['birthdate']),
    birthTown: !exists(json, 'birthTown') ? undefined : json['birthTown'],
    birthState: !exists(json, 'birthState') ? undefined : json['birthState'],
    birthCertificateId: !exists(json, 'birthCertificateId')
      ? undefined
      : json['birthCertificateId'],
    americanIndianOrAlaskaNative: !exists(json, 'americanIndianOrAlaskaNative')
      ? undefined
      : json['americanIndianOrAlaskaNative'],
    asian: !exists(json, 'asian') ? undefined : json['asian'],
    blackOrAfricanAmerican: !exists(json, 'blackOrAfricanAmerican')
      ? undefined
      : json['blackOrAfricanAmerican'],
    nativeHawaiianOrPacificIslander: !exists(
      json,
      'nativeHawaiianOrPacificIslander'
    )
      ? undefined
      : json['nativeHawaiianOrPacificIslander'],
    white: !exists(json, 'white') ? undefined : json['white'],
    hispanicOrLatinxEthnicity: !exists(json, 'hispanicOrLatinxEthnicity')
      ? undefined
      : json['hispanicOrLatinxEthnicity'],
    gender: !exists(json, 'gender')
      ? undefined
      : GenderFromJSON(json['gender']),
    foster: !exists(json, 'foster') ? undefined : json['foster'],
    recievesC4K: !exists(json, 'recievesC4K') ? undefined : json['recievesC4K'],
    family: FamilyFromJSON(json['family']),
    organization: !exists(json, 'organization')
      ? undefined
      : OrganizationFromJSON(json['organization']),
    enrollments: !exists(json, 'enrollments')
      ? undefined
      : (json['enrollments'] as Array<any>).map(EnrollmentFromJSON),
    updateMetaData: !exists(json, 'updateMetaData')
      ? undefined
      : UpdateMetaDataFromJSON(json['updateMetaData']),
  };
}

export function ChildToJSON(value?: Child | null): any {
  if (value === undefined) {
    return undefined;
  }
  if (value === null) {
    return null;
  }
  return {
    id: value.id,
    sasid: value.sasid,
    firstName: value.firstName,
    middleName: value.middleName,
    lastName: value.lastName,
    suffix: value.suffix,
    birthdate:
      value.birthdate === undefined ? undefined : value.birthdate.toISOString(),
    birthTown: value.birthTown,
    birthState: value.birthState,
    birthCertificateId: value.birthCertificateId,
    americanIndianOrAlaskaNative: value.americanIndianOrAlaskaNative,
    asian: value.asian,
    blackOrAfricanAmerican: value.blackOrAfricanAmerican,
    nativeHawaiianOrPacificIslander: value.nativeHawaiianOrPacificIslander,
    white: value.white,
    hispanicOrLatinxEthnicity: value.hispanicOrLatinxEthnicity,
    gender: GenderToJSON(value.gender),
    foster: value.foster,
    recievesC4K: value.recievesC4K,
    family: FamilyToJSON(value.family),
    organization: OrganizationToJSON(value.organization),
    enrollments:
      value.enrollments === undefined
        ? undefined
        : (value.enrollments as Array<any>).map(EnrollmentToJSON),
    updateMetaData: UpdateMetaDataToJSON(value.updateMetaData),
  };
}
