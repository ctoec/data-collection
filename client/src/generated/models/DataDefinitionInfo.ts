/* tslint:disable */
/* eslint-disable */
/**
 * data-collection
 * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
 *
 * The version of the OpenAPI document: 0.0.1
 *
 *
 * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
 * https://openapi-generator.tech
 * Do not edit the class manually.
 */

import { exists, mapValues } from '../runtime';
/**
 *
 * @export
 * @interface DataDefinitionInfo
 */
export interface DataDefinitionInfo {
  /**
   *
   * @type {string}
   * @memberof DataDefinitionInfo
   */
  example: string;
  /**
   *
   * @type {string}
   * @memberof DataDefinitionInfo
   */
  format: string;
  /**
   *
   * @type {string}
   * @memberof DataDefinitionInfo
   */
  reason: string;
  /**
   *
   * @type {string}
   * @memberof DataDefinitionInfo
   */
  definition: string;
  /**
   *
   * @type {string}
   * @memberof DataDefinitionInfo
   */
  required: string;
  /**
   *
   * @type {string}
   * @memberof DataDefinitionInfo
   */
  formattedName: string;
}

export function DataDefinitionInfoFromJSON(json: any): DataDefinitionInfo {
  return DataDefinitionInfoFromJSONTyped(json, false);
}

export function DataDefinitionInfoFromJSONTyped(
  json: any,
  ignoreDiscriminator: boolean
): DataDefinitionInfo {
  if (json === undefined || json === null) {
    return json;
  }
  return {
    example: json['example'],
    format: json['format'],
    reason: json['reason'],
    definition: json['definition'],
    required: json['required'],
    formattedName: json['formattedName'],
  };
}

export function DataDefinitionInfoToJSON(
  value?: DataDefinitionInfo | null
): any {
  if (value === undefined) {
    return undefined;
  }
  if (value === null) {
    return null;
  }
  return {
    example: value.example,
    format: value.format,
    reason: value.reason,
    definition: value.definition,
    required: value.required,
    formattedName: value.formattedName,
  };
}
