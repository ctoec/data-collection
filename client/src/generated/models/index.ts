export * from './Age';
export * from './C4KCertificate';
export * from './CdcReport';
export * from './Child';
export * from './Enrollment';
export * from './EnrollmentReport';
export * from './Family';
export * from './FamilyDetermination';
export * from './FlattenedEnrollment';
export * from './Funding';
export * from './FundingSource';
export * from './FundingSpace';
export * from './FundingTime';
export * from './FundingTimeSplit';
export * from './FundingTimeSplitUtilization';
export * from './Gender';
export * from './Organization';
export * from './OrganizationPermission';
export * from './Region';
export * from './ReportingPeriod';
export * from './Site';
export * from './SitePermission';
export * from './User';
