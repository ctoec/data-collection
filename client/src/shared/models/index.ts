export * from './db';

export * from './AgeGroup';
export * from './BirthCertificateType';
export * from './CareModel';
export * from './ColumnMetadata';
export * from './ExitReason';
export * from './FundingSource';
export * from './FundingSourceTime';
export * from './FundingTime';
export * from './Gender';
export * from './Region';
export * from './ObjectWithValidationErrors';
