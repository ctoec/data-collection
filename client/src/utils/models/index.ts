export * from './getCurrentEnrollment';
export * from './getCurrentFunding';
export * from './childHasEnrollmentsActiveInMonth';
export * from './getShortFundingSourceName';
export * from './getLastEnrollment';
export * from './getLastFunding';
