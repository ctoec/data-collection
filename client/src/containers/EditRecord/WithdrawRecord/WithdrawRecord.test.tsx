import React from 'react';
import {
  snapshotTestHelper,
  accessibilityTestHelper,
} from '../../../testHelpers';
import { WithdrawRecord } from '.';
import { Enrollment } from '../../../shared/models';

const enrollment = {
  site: { siteName: 'Name' },
} as Enrollment;
describe('EditRecord', () => {
  describe('WithdrawRecord', () => {
    snapshotTestHelper(
      <WithdrawRecord
        childName="Name"
        enrollment={enrollment}
        reportingPeriods={[]}
        toggleOpen={jest.fn()}
      />
    );

    accessibilityTestHelper(
      <WithdrawRecord
        childName="Name"
        enrollment={enrollment}
        reportingPeriods={[]}
        toggleOpen={jest.fn()}
      />
    );
  });
});
