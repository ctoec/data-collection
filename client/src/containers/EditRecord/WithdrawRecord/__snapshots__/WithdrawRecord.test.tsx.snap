// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditRecord WithdrawRecord matches snapshot 1`] = `
<div
  aria-modal="true"
  class="ReactModal__Content ReactModal__Content--after-open"
  role="dialog"
  style="position: absolute; top: 50%; left: 50%; right: 40px; border: 1px solid #ccc; background: rgb(255, 255, 255); overflow: auto; border-radius: 4px; outline: none; padding: 20px; transform: translate(-50%, -50%); max-height: 100%; overflow-y: scroll;"
  tabindex="-1"
>
  <div
    class="grid-row display-flex flex-justify flex-align-center"
  >
    <h2
      class="margin-bottom-0"
    >
      Withdraw 
      Name
    </h2>
    <div
      class="oec-modal-close"
    >
      <button
        class="usa-button usa-button--unstyled bx--btn bx--btn--primary"
        tabindex="0"
        type="button"
      >
        <div
          class="display-flex flex-column flex-align-center"
        >
          <svg
            class="height-3 text-base-darker"
            data-icon="times"
            data-prefix="fas"
            fill="currentColor"
            viewBox="0 0 352 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
            />
          </svg>
          <p
            class="margin-y-05"
          >
            close
          </p>
        </div>
      </button>
    </div>
  </div>
  <div
    class=""
  >
    <div
      class="grid-row"
    >
      <div
        class="grid-col"
      >
        <p>
          Name
        </p>
        <p>
          Age: 
        </p>
        <p>
          Enrollment date: 
        </p>
      </div>
      <div
        class="grid-col"
      />
    </div>
    <form
      class="usa-form usa-form"
    >
      <fieldset
        aria-labelledby="fieldset-legend-end-date end-date-hint"
        class="usa-fieldset"
        id="end-date"
      >
        <legend
          id="fieldset-legend-end-date"
        >
          <span
            class="usa-label"
          >
            Enrollment end date
          </span>
        </legend>
        <span
          class="usa-hint text-italic"
          id="end-date-hint"
        >
          For example: 06/21/2008
        </span>
        <div>
          <div
            class="bx--form-item"
          >
            <div
              class="bx--date-picker bx--date-picker--single"
              id="end-date-picker"
            >
              <div
                class="bx--date-picker-container"
              >
                <label
                  class="bx--label bx--visually-hidden"
                  for="end-date-input"
                >
                  Date picker
                </label>
                <div
                  class="bx--date-picker-input__wrapper"
                >
                  <input
                    class="bx--date-picker__input flatpickr-input"
                    id="end-date-input"
                    pattern="\\\\d{1,2}\\\\/\\\\d{1,2}\\\\/\\\\d{4}"
                    placeholder="mm/dd/yyyy"
                    type="text"
                    value=""
                  />
                  <svg
                    aria-hidden="true"
                    class="bx--date-picker__icon"
                    fill="currentColor"
                    focusable="false"
                    height="16"
                    preserveAspectRatio="xMidYMid meet"
                    role="img"
                    viewBox="0 0 32 32"
                    width="16"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M26,4h-4V2h-2v2h-8V2h-2v2H6C4.9,4,4,4.9,4,6v20c0,1.1,0.9,2,2,2h20c1.1,0,2-0.9,2-2V6C28,4.9,27.1,4,26,4z M26,26H6V12h20	V26z M26,10H6V6h4v2h2V6h8v2h2V6h4V10z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
      </fieldset>
      <div
        class="usa-form-group"
      >
        <label
          class="usa-label"
          for="exit-reason"
        >
          <span
            class="text-bold"
          >
            Exit reason
          </span>
        </label>
        <select
          aria-required="true"
          class="usa-select"
          id="exit-reason"
          name=""
        >
          <option
            value=""
          >
            - Select -
          </option>
          <option
            value="Aged out"
          >
            Aged out
          </option>
          <option
            value="Moved within program"
          >
            Moved within program
          </option>
          <option
            value="Stopped attending"
          >
            Stopped attending
          </option>
          <option
            value="Chose to attend a different program"
          >
            Chose to attend a different program
          </option>
          <option
            value=" Moved within Connecticut"
          >
             Moved within Connecticut
          </option>
          <option
            value="Moved to another state"
          >
            Moved to another state
          </option>
          <option
            value="Withdrew due to lack of payment"
          >
            Withdrew due to lack of payment
          </option>
          <option
            value="Child was asked to leave"
          >
            Child was asked to leave
          </option>
          <option
            value="Unknown"
          >
            Unknown
          </option>
        </select>
      </div>
      <button
        class="usa-button usa-button--outline bx--btn bx--btn--primary"
        tabindex="0"
        type="button"
      >
        Cancel
      </button>
      <button
        class="usa-button bx--btn bx--btn--primary"
        tabindex="0"
        type="submit"
      >
        Withdraw
      </button>
    </form>
  </div>
</div>
`;
